src/components/AddProductDialog.vue

```diff
@@ -0,0 +1,125 @@
+<template>
+    <v-dialog
+        v-model="addProductDialogIsActive"
+        fullscreen
+        scrollable
+        transition="dialog-bottom-transition"
+    >
+        <v-card>
+            <v-toolbar
+                dark
+                color="primary"
+            >
+                <v-btn
+                    dark
+                    icon
+                    @click="addProductDialogIsActive = false"
+                >
+                    <v-icon>mdi-close</v-icon>
+                </v-btn>
+                <v-toolbar-title>Add Product</v-toolbar-title>
+                <v-spacer></v-spacer>
+                <v-toolbar-items>
+                    <v-btn>Add Product</v-btn>
+                </v-toolbar-items>
+            </v-toolbar>
+            <div
+                class="d-flex flex-column pa-4"
+            >
+                <v-switch
+                    label="Visible"
+                ></v-switch>
+                <v-text-field
+                    clearable
+                    label="Product Name"
+                >
+                </v-text-field>
+                <v-select
+                    density="compact"
+                    label="Default Product Variant"
+                    :items="['Bar', 'Baz', 'Zep']"
```

Comment:
For the future, I usually think it's a better appraoch to have mock data already as a variable in the script part.
However I guess it's fine for this PR